translator/translate/otel/processor/jmxprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,12 @@
+processors:
+  filter/1:
```

Comment:
It would be better to give a better name here so we can map corresponding  processor to each pipeline and easier in debugging